platform/core/PolyPodCoreAndroid/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
We should really find a way to ignore the entire `.idea` directory - this is first and foremost a Gradle project, Idea/Android Studio is just one possible IDE.